comment_iowriter.go

```diff
@@ -41,7 +42,7 @@ func NewUnifiedCommentWriter(w io.Writer) *UnifiedCommentWriter {
 	return &UnifiedCommentWriter{w: w}
 }
 
-func (mc *UnifiedCommentWriter) Post(c *Comment) error {
+func (mc *UnifiedCommentWriter) Post(_ context.Context, c *Comment) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method UnifiedCommentWriter.Post should have comment or be unexported